e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yml

```diff
@@ -23,35 +23,33 @@
   #  - {role: k8s-openebs-operator-helm}
 
   vars_files:
-    - replica-node-affinity-vars.yaml
+    - replica-node-affinity-vars.yml
 
   tasks:
 
    - block:
 
+       - name: Start logger.
+         include: prerequisistes.yml
+
 # Checking if OpenEBS components are integrated successfully
 
-       - name: 1) Check whether maya-apiserver pod is deployed
-         shell: source ~/.profile; kubectl get pods --all-namespaces | grep maya-apiserver
-         args:
-           executable: /bin/bash
-         register: result
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
-         until: "'Running' in result.stdout"
-         delay: 30
-         retries: 5
-     
-       - name: 1a) Get $HOME of K8s master for kubernetes user
-         shell: source ~/.profile; echo $HOME
-         args:
-           executable: /bin/bash
-         register: result_kube_home
+       - name: 1) Check status of maya-api server
+         include_tasks: "{{utils_path}}/deploy_check.yml"
+         vars:
+            ns: openebs
+            app: maya-api
```

Comment:
Since we are updating the helm to use stable charts, maya-api server is deprecated and now it will be openeb-api server. Can you make the changes accordingly?

Reply:
Done